designs/PackageVulnerability/DotnetListPackageVulnerable.md

```diff
@@ -34,13 +34,12 @@ Combining the `--vulnerable` command option with the following flags will not be
 
 * `--outdated`
 * `--deprecated`
-* `--offline`
 
-The following flags will be ignored (but not error) when combined with the `--vulnerable` command option:
+The following flags will are only relevant to the `--outdated` command option, or any option that potentially marks versions with an `(O)` marker. They will continue to be ignored, without erroring, when using the `--deprecated` option, which is existing behavior. They will now also be ignored without erroring when using the `--vulnerable` option:
 
-* `--include-prerelease` (currently requires `--outdated` option)
-* `--highest-minor` (currently requires `--outdated` option)
-* `--highest-patch` (currently requires `--outdated` option)
+* `--include-prerelease`
+* `--highest-minor`
+* `--highest-patch`
 
 An informational message will be printed when the ignored command options are used:
```

Comment:
For the information message below, can/should we mention why the options were ignored? Consider:

```shell
	The command option(s) '--include-prerelease', '--highest-minor', and '--highest-patch' were ignored as they are only relevant to the '--outdated' command option.
```

Reply:
Per @JonDouglas's comment above, we'll remove footers for now and let customers use `--help` for the full help experience.